### PR TITLE
Add parameter uncertainty vignette

### PR DESCRIPTION
**Pull request by pratikunterwegs**
This PR adds a vignette showing how to use {daedalus} to model the effect of parameter uncertainty on pandemic cost projections.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -30,9 +30,9 @@ _daedalus_ implements the integrated epidemiological and economic model in @haw2
 
 You can install the development version of daedalus from [GitHub](https://github.com/) using the _remotes_ package.
 
-``` r
-# install.packages("remotes")
-remotes::install_github("j-idea/daedalus", upgrade = FALSE)
+```r
+# install.packages("pak")
+pak::pak("jameel-institute/daedalus", upgrade = FALSE)
```
Comment (richfitz) [inline README.Rmd:35]: I suggest adding

```
install.packages('daedalus', repos = c('https://jameel-institute.r-universe.dev', 'https://cloud.r-project.org'))
```

Formatted however you prefer as an option too; this usually ends up going better on windows machines than other methods

File: vignettes/parameter_uncertainty.Rmd [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+**Note that** running `daedalus()` iteratively may take some time, as the function checks the inputs and internally prepares parameters each time, as well as preparing the output data. Future package versions aim to streamline these steps and provide a stripped down version of the function optimised for scenario modelling for parameter fitting or with parameter uncertainty.
+
+```{r uncertainty_run_model}
+# run daedalus()
+output_list <- lapply(
+  infection_list, daedalus,
+  country = "Norway"
+)
+```
+
```
Comment (richfitz) [inline vignettes/parameter_uncertainty.Rmd:96]: Suggest that if people want to run this in parallel they can use a framework of their choice, e.g, furrr?

File: vignettes/parameter_uncertainty.Rmd [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+```{r}
+# get costs
+costs <- vapply(
+  output_list, get_costs, numeric(1), "total"
+)
+```
+
+We can plot the total costs to view the distribution using the [_ggdist_ package](https://cran.r-project.org/package=ggdist) to visualise the distribution.
+
+```{r plot_total_costs_dist, echo=FALSE}
```
Comment (richfitz) [inline vignettes/parameter_uncertainty.Rmd:110]: Suggest echoing here too so people can start building plots with the code you have working

File: vignettes/parameter_uncertainty.Rmd [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+First, we access the SARS 2004 $R_0$ provided in _daedalus_, and using this as a mean we draw 100 samples from a normal distribution with a standard deviation of 0.1.
+
+```{r get_sars_r0}
+# get SARS-1 R0
+sars_2004 <- daedalus_infection("sars_cov_1")
+sars_2004_r0 <- get_data(sars_2004, "r0")
+
+# draw samples from a normal distribution
+r0_samples <- withr::with_seed(
```
Comment (richfitz) [inline vignettes/parameter_uncertainty.Rmd:55]: I don't think we should be encouraging users to set the seed generally. If you want to make this deterministic, I suggest putting `set.seed(1)` in the setup block instead

**Review by richfitz — CHANGES_REQUESTED** [review]

File: README.Rmd (hunk 1 above)
Comment (pratikunterwegs) [reply README.Rmd:35]: Yep will do, forgot we have this.

File: vignettes/parameter_uncertainty.Rmd (hunk 3 above)
Comment (pratikunterwegs) [reply vignettes/parameter_uncertainty.Rmd:110]: Sure can do, I usually hide plotting code as it tends to get pretty long.

**Comment by pratikunterwegs** [comment]
I imagine the other workflows will pass as well.

**Review by richfitz — APPROVED** [review]

**Pull request merged**